### PR TITLE
add r-codemetar recipe

### PR DESCRIPTION
**Pull request by luizirber**
Generated with conda_r_skeleton_helper

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/r-codemetar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    as detailed at <https://codemeta.github.io>. This package provides utilities to
+    generate, parse, and modify 'codemeta.json' files automatically for R packages,
+    as well as tools and examples for working with 'codemeta.json' 'JSON-LD' more generally.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
```
Comment (luizirber) [inline recipes/r-codemetar/meta.yaml]: shoudl I check if `conda-forge/r` is OK with being a maintainer?

File: recipes/r-codemetar/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    - r-whisker
+    - r-xml2
+
+test:
+  commands:
+    - $R -e "library('codemetar')"           # [not win]
+    - "\"%R%\" -e \"library('codemetar')\""  # [win]
+
+about:
+  home: https://github.com/ropensci/codemetar, https://ropensci.github.io/codemetar
```
Comment (luizirber) [inline recipes/r-codemetar/meta.yaml]: linter didn't complain, but is it OK to have two URLs?

**Comment by luizirber** [comment]
Ready for review and merge @conda-forge/help-r

File: recipes/r-codemetar/meta.yaml (hunk 1 above)
Comment (cbrueffer) [reply recipes/r-codemetar/meta.yaml]: No need, we're all in by default ;-)

**Pull request merged**